integration/config/config_test.go

```diff
@@ -0,0 +1,138 @@
+/*
+Copyright IBM Corp All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package config
+
+import (
+	"io/ioutil"
+	"os"
+	"syscall"
+
+	docker "github.com/fsouza/go-dockerclient"
+	"github.com/golang/protobuf/proto"
+	"github.com/hyperledger/fabric/integration/nwo"
+	"github.com/hyperledger/fabric/integration/nwo/commands"
+	"github.com/hyperledger/fabric/pkg/config"
+	"github.com/tedsuo/ifrit"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("Config", func() {
+	var (
+		client  *docker.Client
+		testDir string
+		network *nwo.Network
+		process ifrit.Process
+	)
+
+	BeforeEach(func() {
+		var err error
+		testDir, err = ioutil.TempDir("", "config")
+		Expect(err).NotTo(HaveOccurred())
+
+		client, err = docker.NewClientFromEnv()
+		Expect(err).NotTo(HaveOccurred())
+
+		network = nwo.New(nwo.BasicSolo(), testDir, client, StartPort(), components)
+		network.UseConfigPkg = true
```

Comment:
This is the kind of switch I wanted to avoid introducing to `nwo`. The primitives should be in place such that this test can simply generate the config transaction without requiring any other changes (outside of port allocation).

Reply:
Done